### PR TITLE
Featue/country select

### PR DESCRIPTION
**Pull request by mateuszkula**
I want to merge this change because...

It adds new Select component:
obraz

### CODE REVIEW

File: src/@next/components/atoms/Select/test.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    "Russia",
+    "China",
+    "Croatia",
+    "Spain",
+    "Canada",
+  ],
+};
+
+describe("<Select />", () => {
+  // Example test
```
Comment (bogdal) [inline src/@next/components/atoms/Select/test.tsx]: It looks like we don't need this comment.

File: src/@next/components/atoms/Select/Select.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      activeOptionRef.current.focus();
+    }
+  }, [isOpen]);
+
+  const saveValue = (option: string) => {
+    setOpen(false);
+    setValue(option);
+  };
+
+  const shouldFilter = value !== searchValue;
```
Comment (jwm0) [inline src/@next/components/atoms/Select/Select.tsx]: please no 😄

File: src/@next/components/atoms/Select/Select.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  const filteredOptions = filterOptions(searchValue, options, shouldFilter);
+
+  return (
+    <S.Wrapper
+      onClick={() => {
+        setOpen(!isOpen);
+        setSearchValue(value);
```
Comment (jwm0) [inline src/@next/components/atoms/Select/Select.tsx]: Setting search value equal to current value here is bad UX. I'd get rid off it completely and use Input as follows:
```
<Input
  placeholder={value}
  value={isOpen ? searchValue : value}
```
This way, before a user inputs anything he still gets the original `value` but he starts from an empty string

File: src/@next/components/atoms/Select/Select.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+      }}
+      ref={setElementRef()}
+    >
+      <Input
+        label={label}
+        value={isOpen ? searchValue : value}
+        onChange={e => {
+          setOpen(true);
```
Comment (jwm0) [inline src/@next/components/atoms/Select/Select.tsx]: is there a case where onChange is fired while select box is closed?

**Comment by mateuszkula** [comment]
Im closing this PR as we will use react-select for country select as this implementation was missing keyboard support

**Pull request closed without merging**